applications/NXarpes.nxdl.xml

```diff
@@ -21,134 +21,113 @@
 #
 # For further information, see http://www.nexusformat.org
 -->
-<definition xmlns="http://definition.nexusformat.org/nxdl/3.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" category="application" type="group" name="NXarpes" extends="NXobject" xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd">
-    <doc>
-         This is an application definition for angular resolved photo electron spectroscopy.
-         
-         It has been drawn up with hemispherical electron analysers in mind.
-         
-         This definition is a legacy support for older NXarpes experiments.
-         There is, however, a newer definition to collect data &amp; metadata
-         for general photoemission experiments, called :ref:NXmpes,
-         as well as a specialization for ARPES experiments, called :ref:NXmpes_arpes."
-    </doc>
-    <group type="NXentry">
-        <attribute name="entry">
-            <doc>
-                 NeXus convention is to use "entry1", "entry2", ...
-                 for analysis software to locate each entry.
-            </doc>
-        </attribute>
-        <field name="title" type="NX_CHAR"/>
-        <field name="start_time" type="NX_DATE_TIME"/>
-        <field name="definition">
-            <doc>
-                 Official NeXus NXDL schema to which this file conforms.
-            </doc>
-            <enumeration>
-                <item value="NXarpes"/>
-            </enumeration>
+<definition name="NXarpes" extends="NXobject" type="group"
+	    category="application"
```

Comment:
Strange new lines. Note that it does not seem to match
https://github.com/nexusformat/definitions/blob/main/applications/NXarpes.nxdl.xml